### PR TITLE
LPS-100144 Add default label to AssetTagsSelector

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

**Comment by carloslancha** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
Hi @john-co we're getting some poshi errors 'cause an element is not being found (but I can see it there):
```
Element is not present at "//div[contains(@id,'assetTagsSelector')]//input[@class='form-control-inset']"
```

Could you please take a look? Thanks!

**Comment by john-co** [comment]
Just started reviewing :)

**Comment by john-co** [comment]
ci:test:bundle

**Comment by john-co** [comment]
ci:test:relevant

**Comment by john-co** [comment]
Hi @carloslancha,

I'm able to reproduce the issue on Chromium 65.0 with this test bundle. Chrome 65 is the browser version our CI runs on.
TagsFieldNotVisible

image

The issue is not reproducible on Chrome 73. Would you be able to include a fix for CI testing?

File: modules/apps/frontend-taglib/frontend-taglib-clay/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
 		"@clayui/loading-indicator": "3.0.0",
 		"@clayui/management-toolbar": "3.0.0-alpha.1",
 		"@clayui/modal": "3.0.0",
-		"@clayui/multi-select": "3.0.0-alpha.1",
+		"@clayui/multi-select": "3.0.0",
```
Comment (jbalsas) [inline modules/apps/frontend-taglib/frontend-taglib-clay/package.json]: We need to wait on this change until we've properly updated our `imports` config in our bundler preset. @matuzalemsteles said we might release today... 🤞

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/wincent/liferay-portal/pull/72. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/5521e0ca6b24~7...5521e0ca6b24) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**